### PR TITLE
History and like single video page

### PR DESCRIPTION
**Pull request by varun-Tad**
Please review the following files

1.LikeDislike folder
2. Singlepage folder
3. Historypage folder

Features tor review:

1. Clicking on any video in explore section will take you to a single-page video
2. Like and unlike video
3. history update on clicking the videos

Preview: https://gleeful-daifuku-ed57ad.netlify.app/

### CODE REVIEW

File: dressvid/src/routes/Historypage/Historypage.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+.history-clearBtn button {
+  cursor: pointer;
+  border-radius: 1rem;
+  padding: 0.7rem;
+  border: 0.3px solid #848482;
+  font-weight: 600;
+}
+
+.history-clearBtn button:hover {
+  background-color: #848482;
```
Comment (anamkaa) [inline dressvid/src/routes/Historypage/Historypage.css:79]: Can use css variables here

File: dressvid/src/routes/Singlepage/Singlepage.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+.video-name {
+  margin-top: 6px;
+  margin-bottom: 0;
+}
+
+.footer-left-para {
+  font-size: 1rem;
+  color: #606060;
```
Comment (anamkaa) [inline dressvid/src/routes/Singlepage/Singlepage.css:33]: using variables will make it reusable

**Review by anamkaa — APPROVED** [review]
Suggested some minor changes , other than that,  good naming convention used and clean code.

**Review by shubhamkumar648 — APPROVED** [review]
already mention for changes , remaining all goods pr are ready to merge

**Pull request merged**